packages/vitest/src/defaults.ts

```diff
@@ -2,7 +2,7 @@ import type { BenchmarkUserOptions, ResolvedCoverageOptions, UserConfig } from '
 import { isCI } from './utils/env'
 
 export const defaultInclude = ['**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']
-export const defaultExclude = ['**/node_modules/**', '**/dist/**', '**/cypress/**', '**/.{idea,git,cache,output,temp}/**', '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*']
+export const defaultExclude = ['**/{node_modules,dist,cypress,tests,e2e-tests}/**', '**/.{idea,git,cache,output,temp}/**', '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*']
```

Comment:
Why does it exclude test folder now?

Reply:
The explanation is in https://github.com/vitest-dev/vitest/issues/3102

Reply:
How exactly is it a good choice to exclude every test?

Reply:
It's not excluding every test, it's excluding the tests folder, because unit tests are not usually grouped in a folder, but placed next to the file they test, unlike e2e tests. As I explained in the issue:

> Since a generally accepted good practice* is to put unit tests next to the code they test, and e2e tests in a separate folder, excluding such a folder would be a better default setting.
> 
> In many projects I see that all the configuration file does is modify include or exclude to exclude e2e tests.

Reply:
I don't think this is true. Even in Vitest tests are grouped inside a tests folder

Reply:
@sheremet-va Alright. I have removed that path. Please see my comment below. Thank you!